client_proposal.go

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2020 Chair of Applied Cryptography, Technische Universität
+// Darmstadt, Germany. All rights reserved. This file is part of
+// perun-eth-mobile. Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package prnm
+
+import (
+	ethwallet "perun.network/go-perun/backend/ethereum/wallet"
```

Comment:
Since we often deal with `go-perun/wallet` and `go-perun/backend/ethereum/wallet` imports simultaneously, it is more clearly to always name the ethereum packages `ethwallet` or `ethchannel`. Otherwise you always first have to check in the file header whether `channel` is the core or eth channel package etc.